core/src/test/scala/org/genivi/sota/core/CampaignResourceSpec.scala

```diff
@@ -146,6 +168,18 @@ class CampaignResourceSpec extends FunSuite
       status shouldBe expectedCode
     }
 
+  def setDeltaFrom(id: Campaign.Id, pkg: Option[PackageId] = None, expectedCode: StatusCode): Unit = {
```

Comment:
Why the default argument? Since it is in the middle you can't default anyway? So either remove the `= None` or make `pkg` be the last argument.